cms/static/js/views/course_info_update.js

```diff
@@ -257,14 +277,14 @@ define(['js/views/validation', 'codemirror', 'js/models/course_update',
                 else {
                 // close the modal and insert the appropriate data
                     this.$currentPost.removeClass('editing');
-                    this.$currentPost.find('.date-display').html(targetModel.get('date'));
+                    this.$currentPost.find('.date-display').text(targetModel.get('date'));
                     this.$currentPost.find('.date').val(targetModel.get('date'));
 
                     var content = CourseInfoHelper.changeContentToPreview(
                     targetModel, 'content', this.options['base_asset_url']);
                     try {
                     // just in case the content causes an error (embedded js errors)
-                        this.$currentPost.find('.update-contents').html(content);
+                        HtmlUtils.setHtml(this.$currentPost.find('.update-contents'), HtmlUtils.HTML(content));
```

Comment:
Where does ``content`` come from? It would be better to wrap it with ``HTML`` immediately, rather than have consumers remember to do it like this. It makes it clearer that it is HTML at the source. You might also want to name it as ``fooHtml`` (although ``contentHtml`` sounds weird) so that code that consumes it is clear that it contains safely escaped HTML.